lib/models/transaction.dart

```diff
@@ -1,33 +1,38 @@
 class Transaction implements Comparable {
-  final int timestamp;
+  final String createdAt;
   final double amount;
-  final String goalName;
-  final String profileGuid;
+  final String destinationName;
+  final String parentGuid;
 
   Transaction(
-      {required this.timestamp,
+      {required this.createdAt,
       required amount,
-      required this.profileGuid,
-       this.goalName = "Christ Pres Chruch -b"})
+      required this.parentGuid,
+      this.destinationName = "Christ Pres Chruch -b"})
```

Comment:
Should this still be here? There should always be a destination name right? :)

Reply:
I agree, need to be required as others